src/lib/TranscriptEditor/index.js

```diff
@@ -52,43 +62,51 @@ class TranscriptEditor extends React.Component {
   componentDidUpdate(prevProps, prevState) {
     // Transcript and media passed to component at same time
     if (
-      (prevState.transcriptData !== this.state.transcriptData)
-        && (prevProps.mediaUrl !== this.props.mediaUrl )
+      prevState.transcriptData !== this.state.transcriptData &&
+      prevProps.mediaUrl !== this.props.mediaUrl
     ) {
       console.info('Transcript and media');
       this.ifPresentRetrieveTranscriptFromLocalStorage();
     }
     // Transcript first and then media passed to component
     else if (
-      (prevState.transcriptData === this.state.transcriptData)
-      && (prevProps.mediaUrl !== this.props.mediaUrl)
+      prevState.transcriptData === this.state.transcriptData &&
+      prevProps.mediaUrl !== this.props.mediaUrl
     ) {
       console.info('Transcript first and then media');
       this.ifPresentRetrieveTranscriptFromLocalStorage();
     }
     // Media first and then transcript passed to component
     else if (
-      (prevState.transcriptData !== this.state.transcriptData)
-      && (prevProps.mediaUrl === this.props.mediaUrl)
+      prevState.transcriptData !== this.state.transcriptData &&
+      prevProps.mediaUrl === this.props.mediaUrl
     ) {
       console.info('Media first and then transcript');
       this.ifPresentRetrieveTranscriptFromLocalStorage();
     }
   }
 
   ifPresentRetrieveTranscriptFromLocalStorage = () => {
-    if (this.timedTextEditorRef.current !== undefined) {
-      if (this.timedTextEditorRef.current.isPresentInLocalStorage(this.props.mediaUrl)) {
+    console.log('checking - ifPresentRetrieveTranscriptFromLocalStorage');
+    if (
+      this.timedTextEditorRef !== undefined &&
```

Comment:
I think this can just be written as

```
const editor = this.timedTextEditorRef;

if (editor && editor.current) {
  if (editor.current.isPresentInLocalStorage(this.props.mediaUrl)) {
     console.info('Already present in local storage.');
     this.timedTextEditorRef.current.loadLocalSavedData(this.props.mediaUrl);	        
  } else {
    console.info('Not present in local storage.');
  }

}

```

Reply:
did you try it? does it work?

Reply:
ok trying to change to 

```js
 ifPresentRetrieveTranscriptFromLocalStorage = () => {
    const timedTextEditor = this.timedTextEditorRef;
    if (timedTextEditor && timedTextEditor.current) {
      if (timedTextEditor.current.isPresentInLocalStorage(this.props.mediaUrl)) {
        console.info('Already present in local storage.');
        timedTextEditor.current.loadLocalSavedData(this.props.mediaUrl);
      } else {
        console.info('Not present in local storage.');
      }
    }
  };
```

I prefer `timedTextEditor` coz is less ambiguous compared to `editor`.

Reply:
ok, tested it works